### PR TITLE
Add examples for run:yeoman scaffolder action and improve related tests

### PR DESCRIPTION
**Pull request by parmar-abhinav**
## Hey, I just made a Pull Request!
Contribution of run:yeoman  examples as part of https://github.com/backstage/backstage/issues/18832

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/tasty-wombats-exercise.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend-module-yeoman': patch
+---
+
+Add examples for run:yeoman scaffolder action and improve related tests
```
Comment (vinzscam) [inline .changeset/tasty-wombats-exercise.md:5]:
```suggestion
Add examples for `run:yeoman` scaffolder action.
```

**Review by vinzscam — CHANGES_REQUESTED** [review]
nice!

File: .changeset/tasty-wombats-exercise.md (hunk 1 above)
Comment (parmar-abhinav) [reply .changeset/tasty-wombats-exercise.md]: Thank you for reviewing the PR. Suggested changes are done. Please check.

Comment (vinzscam) [reply .changeset/tasty-wombats-exercise.md]: I think you need to run `yarn` and commit the lockfile as well

Comment (parmar-abhinav) [reply .changeset/tasty-wombats-exercise.md]: Do I need to run `yarn install` inside `backstage/plugins/scaffolder-backend-module-yeoman`  directory. As there is no lock file present inside it.

Comment (parmar-abhinav) [reply .changeset/tasty-wombats-exercise.md]: Since yaml was not present in the dependencies, So I added the yaml dependecy in package.json

Comment (parmar-abhinav) [reply .changeset/tasty-wombats-exercise.md]: Done, please check

**Review by freben — APPROVED** [review]
nice!

**Comment by parmar-abhinav** [comment]
@vinzscam Please review.

**Comment by awanlin** [comment]
Hi @parmar-abhinav, there are changes in this PR that should not be there based on your title and description. Can you please take a look? Might just be a clean rebase needed 👍

**Comment by parmar-abhinav** [comment]
> Hi @parmar-abhinav, there are changes in this PR that should not be there based on your title and description. Can you please take a look? Might just be a clean rebase needed 👍

Thank you for your feedback, Updated the PR please check.

**Review by awanlin — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]
:shipit: 👍

**Pull request merged**